package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "omt-odt-language",
-    "version": "2.6.1",
+    "version": "2.6.2",
```

Comment:
This change is a breaking change. I think its better to bump the version to 3.0.0

Reply:
After discussed its solved

Reply:
Discussed and agreed to leave at 2.6.2, because there are no functional changes